conans/test/functional/toolchains/cmake/cmakedeps/test_apple_frameworks.py

```diff
@@ -74,7 +74,7 @@ class AppleframeworkConan(ConanFile):
 
                 def build(self):
                     cmake = CMake(self)
-                    cmake.configure(source_folder="src")
+                    cmake.configure(build_script_folder="src")
```

Comment:
this test is confusing, abusing the ``build_script_folder``, instead of defining the correct ``layout`` with the correct ``source``, which is the case. I think that it is ok, but it would be worth adding a comment to say it is known, and this is not the intended usage.